packages/flutter_tools/templates/package/.gitignore.tmpl

```diff
@@ -56,7 +56,7 @@ build/
 **/ios/**/xcuserdata
 **/ios/.generated/
 **/ios/Flutter/App.framework
-**/ios/Flutter/Flutter.framework
+**/ios/Flutter/Flutter.*
```

Comment:
The Flutter.podspec is now getting copied in.  People will have to add it to their .gitignore or check it in... ☹️